### PR TITLE
Chore / Upgrade to Next.js 12.2

### PR DESCRIPTION
**Pull request by robinandeer**
## Describe your changes

Update to stable middleware.

## Justify why they are needed

Keep up-to-date

### CODE REVIEW

**Comment by robinandeer** [comment]
Current dependencies on/for this PR:
* main
  * **PR #260** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/260" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/260?utm_source=stack-comment).

File: turbo.json [hunk 1]
```diff
@@ -33,11 +33,12 @@
...
     },
     "market-web#build": {
-      "outputs": [".next/**"]
+      "outputs": [".next/**"],
+      "dependsOn": ["^build", "$STORYBLOK_API_KEY", "$STORYBLOK_PREVIEW_SECRET"]
     },
     "onboarding#test": {
       "outputs": ["covereage/**"],
-      "dependsOn": ["build"]
+      "dependsOn": ["^build"]
```
Comment (robinandeer) [inline turbo.json:41]: This fixes the issue where we ran the build command for ALL packages rather than just the ones needed for market-web :)

**Review by gustaveen — APPROVED** [review]
👍

**Comment by robinandeer** [comment]
**Graphite Merge Job**

Current status: 🔄 Waiting for checks to pass

This pull request is currently waiting for CI and mergeability checks to pass before being merged.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/260?utm_source=stack-comment).


Job Reference: Fh5gvOSg8Jg6yIMEIGCf

**Pull request merged**